index.bs

```diff
@@ -357,10 +357,14 @@ The presentation times of AV1 samples are given by the ISOBMFF structures. <asse
 
 <assert>The sample entry SHOULD contain a 'colr' box with a colour_type set to 'nclx'</assert>. <assert>If present, the values of colour_primaries, transfer_characteristics, and matrix_coefficients SHALL match the values given in the [=Sequence Header OBU=] (in the configOBUs field or in the associated samples) if the color_description_present_flag is set to 1</assert>. Similarly, <assert>the full_range_flag in the 'colr' box SHALL match the color_range flag in the [=Sequence Header OBU=]</assert>. <assert>When configOBUs does not contain a [=Sequence Header OBU=], this box with colour_type set to 'nclx' SHALL be present.</assert>
 
-<assert>For sample entries corresponding to HDR content, the MasteringDisplayColourVolumeBox 'mdcv' and ContentLightLevelBox 'clli' SHOULD be present</assert>, and <assert>their values SHALL match the values contained in the [=Metadata OBUs=] of type METADATA_TYPE_HDR_CLL and METADATA_TYPE_HDR_MDCV, if present (in the configOBUs or in the samples)</assert>.
+For sample entries corresponding to HDR content, the following applies:
+- <assert>The ContentLightLevelBox 'clli' SHOULD be present</assert>. <assert>Its values SHALL equal the values contained in the [=Metadata OBU=] of type METADATA_TYPE_HDR_CLL, if present (in the configOBUs or in the samples)</assert>.
+- <assert>The MasteringDisplayColourVolumeBox 'mdcv' SHOULD be present</assert>. <assert>Its decoded values SHALL equal (except for precision issues, see the second note below) the decoded values contained in the [=Metadata OBU=] of type METADATA_TYPE_HDR_MDCV, if present (in the configOBUs or in the samples)</assert>.
 
 NOTE: The MasteringDisplayColourVolumeBox 'mdcv' and ContentLightLevelBox 'clli' have identical syntax to the SMPTE2086MasteringDisplayMetadataBox 'SmDm' and ContentLightLevelBox 'CoLL', except that they are of type <code>Box</code> and not <code>FullBox</code>. However, the semantics of the MasteringDisplayColourVolumeBox and SMPTE2086MasteringDisplayMetadataBox have important differences and SHOULD not be confused.
 
+NOTE: In the MasteringDisplayColourVolumeBox 'mdcv', the precison and coded values of the fields are different from the precision and coded values of the corresponding fields in the [=Metadata OBU=] of type METADATA_TYPE_HDR_MDCV. Because of the different precisions, some values can be represented exactly in the MasteringDisplayColourVolumeBox 'mdcv' but not in the [=Metadata OBU=] of type METADATA_TYPE_HDR_MDCV, and vice versa.
+
```

Comment:
```suggestion
NOTE: In the MasteringDisplayColourVolumeBox 'mdcv', the representation and coding of some fields are different from the representation and coding of the corresponding fields in the [=Metadata OBU=] of type METADATA_TYPE_HDR_MDCV. Because of these differences, some values coded in the MasteringDisplayColourVolumeBox 'mdcv' may not have the same precision in the [=Metadata OBU=] of type METADATA_TYPE_HDR_MDCV, and vice versa.
```